src/main/java/org/broadinstitute/hellbender/cmdline/CommandLineProgram.java

```diff
@@ -145,10 +151,11 @@ public Object instanceMain(final String[] argv) {
         } finally {
             // Emit the time even if program throws
             if (!QUIET) {
-                final Date endDate = new Date();
-                final double elapsedMinutes = (endDate.getTime() - startDate.getTime()) / (1000d * 60d);
+                final ZonedDateTime endDateTime = ZonedDateTime.now();
+                final double elapsedMinutes = (Duration.between(startDateTime, endDateTime).toMillis()) / (1000d * 60d);
```

Comment:
Sorry, nvm.  Continue doing what you're doing, that would drop the fractional minutes.